src/forms/templates/utils/index.js

```diff
@@ -1,4 +1,4 @@
-export const RJSF_SEPARATOR = "$";
+export const RJSF_SEPARATOR = "!";
```

Comment:
lets replace with `$$__$$` - having a combination of characters makes it harder to conflict with field IDs, also it is clearer to read (e.g. on the url)

Reply:
Agreed to replace it with `::` which is more clean and manageable